api/v1beta2/imagepolicy_types.go

```diff
@@ -105,6 +105,10 @@ type ImagePolicyStatus struct {
 	// the image repository, when filtered and ordered according to
 	// the policy.
 	LatestImage string `json:"latestImage,omitempty"`
+	// LatestDigest is the digest of the latest image stored in the
+	// accompanying LatestImage field.
+	// +optional
+	LatestDigest string `json:"latestDigest,omitempty"`
```

Comment:
If this is a logical pair of values together with `LatestImage`, it might be better to actually pair them in a struct?

Reply:
It would, but that may actually be for the better?

One thing that I am wondering about at present for example is what happens when the `ImageRepositoryRef` is switched to a different target repository (either by changing the reference, or changes within the reference).

I think at present, for a brief moment of time there may be an advertisement of metadata not necessarily tied to the then repository. Reflecting the data in full would improve this.

Reply:
Just to be sure: Are we ready to introduce a new API version "just" for this change?

Reply:
@hiddeco please see https://github.com/fluxcd/image-reflector-controller/pull/368/commits/36e46d6e1eafe6eb0e08e35d14f239d8d870e71b where I paired all image reference values in a new type and let me know what you think.